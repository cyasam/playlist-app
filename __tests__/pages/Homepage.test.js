import React from 'react'
import { shallow } from 'enzyme'
import { Homepage } from '../../src/scripts/pages/Homepage'

describe('Searchbox Component', () => {
  const setup = (propOverrides) => {
    const props = {
      searchResult: {
        isfetching: false,
        response: [
          {
            id: 'lZoA5ZX4wC0',
            title: 'Video Title',
            description: 'Lorem ipsum dolor sit amed.',
            thubmnail: 'video-thumbnail.jpg'
          },
          
          {
            id: 'lZoA5ZX4wC01',
            title: 'Video Title 2',
            description: 'Lorem ipsum dolor sit amed.',
            thubmnail: 'video-thumbnail-2.jpg'
          }
        ]
      },
      ...propOverrides
    }
    const wrapper = shallow(<Homepage { ...props } />)
    return {
      wrapper,
      props
    }
  }
  
  it('renders properly', () => {
    const { wrapper } = setup()
    expect(wrapper).toMatchSnapshot()
  })
})