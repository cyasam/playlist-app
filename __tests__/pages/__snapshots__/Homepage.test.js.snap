// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbox Component renders properly 1`] = `
<div
  className="container"
>
  <h2>
    Trending Videos
  </h2>
  <VideoList
    videos={
      Array [
        Object {
          "description": "Lorem ipsum dolor sit amed.",
          "id": "lZoA5ZX4wC0",
          "thubmnail": "video-thumbnail.jpg",
          "title": "Video Title",
        },
        Object {
          "description": "Lorem ipsum dolor sit amed.",
          "id": "lZoA5ZX4wC01",
          "thubmnail": "video-thumbnail-2.jpg",
          "title": "Video Title 2",
        },
      ]
    }
  />
</div>
`;
